Tasks/PublishPipelineArtifactV1/task.loc.json

```diff
@@ -38,6 +38,48 @@
       "defaultValue": "",
       "required": false,
       "helpMarkDown": "ms-resource:loc.input.help.artifactName"
+    },
+    {
+      "name": "artifactType",
+      "aliases": [
+        "publishLocation"
+      ],
+      "type": "pickList",
+      "label": "ms-resource:loc.input.label.artifactType",
+      "defaultValue": "pipeline",
+      "required": true,
+      "helpMarkDown": "ms-resource:loc.input.help.artifactType",
+      "options": {
+        "pipeline": "Azure Pipelines",
+        "filepath": "A file share"
+      }
+    },
+    {
+      "name": "fileSharePath",
+      "type": "string",
+      "label": "ms-resource:loc.input.label.fileSharePath",
+      "defaultValue": "",
+      "required": true,
+      "helpMarkDown": "ms-resource:loc.input.help.fileSharePath",
+      "visibleRule": "artifactType = filepath"
+    },
+    {
+      "name": "parallel",
```

Comment:
Could we simplify this to just a single `parallel` (numeric) input? Default value is 1 (or whatever) ... and users can increase it as needed. Btw, how would a user know what to set this to? Why wouldn't they just set it to 512 or something? Feels like a super-advanced property (guessing it existed on the old task)

Reply:
Yeah this is how build artifact is doing currently. There is a description saying "Enter the degree of parallelism, or number of threads used, to perform the copy. The value must be at least 1 and not greater than 128." I think the original idea is to hide the parallel copy option unless a user specify that they want to use parallel copy option, the input field will show up on the UI.